README.md

```diff
@@ -89,38 +89,50 @@ This table captures the minimum memory requirements for our different recipes us
 torchtune supports fine-tuning for the Llama3 8B and 70B models. We currently support LoRA, QLoRA and Full-finetune on a single GPU as well as LoRA and Full fine-tune on multiple devices for the 8B model, and LoRA on multiple devices for the 70B model. For all the details, take a look at our [tutorial](https://pytorch.org/torchtune/stable/tutorials/llama3.html).
 
 
-In our initial experiments for Llama3-8B, QLoRA has a peak allocated memory of ``~9GB`` while LoRA on a single GPU has a peak allocated memory of ``~19GB``. To get started, you can use our default configs to kick off training.
+In our initial experiments for Llama3-8B, QLoRA has a peak allocated memory of ``~9GB`` while LoRA on a single GPU has a peak allocated memory of ``~19GB``. To get started, you can use our default configs to kick off training. 
 
-- 8B LoRA on a single GPU.
+### Single GPU
+
+Full 8B
+
+```bash
+tune run full_finetune_single_device --config llama3/8B_full_single_device
+```
+
+LoRA 8B
 
 ```bash
 tune run lora_finetune_single_device --config llama3/8B_lora_single_device
 ```
 
-- 8B QLoRA on a single GPU
+QLoRA 8B
 
 ```bash
 tune run lora_finetune_single_device --config llama3/8B_qlora_single_device
 ```
 
-- 8B LoRA on 2 GPUs
+### Multi GPU
+
+Full 8B
 
 ```bash
-tune run --nproc_per_node 4 lora_finetune_distributed --config llama3/8B_lora
+tune run --nproc_per_node 2 full_finetune_distributed --config llama3/8B_full
```

Comment:
Actually while you're in here can you set the default for full to run on 4 devices and the default for LoRA to run on 2 devices? That way it will match our recommended usage in the configs [here](https://github.com/pytorch/torchtune/blob/9b2be355a9d0b5a5bcd917cffb55812635763a74/recipes/configs/llama3/8B_full.yaml#L9) and [here](https://github.com/pytorch/torchtune/blob/9b2be355a9d0b5a5bcd917cffb55812635763a74/recipes/configs/llama3/8B_lora.yaml#L9).

Reply:
Done